### ISSUE TITLE
pietsmiet.de: breakage

### ISSUE THREAD

**Issue by Gamer10222** [uBlockOrigin/uAssets#15604]
### Prerequisites

- [X] I read and understood the [policy as to what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets)
- [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out)
- [X] I forced an update of my filter lists (**how to do this:** [click _"Purge all caches"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches), then [click _"Update now"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now))
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I verified that the issue is not caused by removing any of the default lists
- [X] I did not enable external filter lists, or I verified that the issue still occurs without enabling external filter lists
- [X] I do not have custom filters/rules, or I verified that the issue still occurs without custom filters/rules
- [X] I am not using [another content blocker along uBO](https://twitter.com/gorhill/status/1033706103782170625)
- [X] I verified the issue is not caused by the browser built-in blocker, or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through VPN)
- [ ] I did not answer truthfully to **all** the above checkpoints

### URL address of the web page

`https://www.pietsmiet.de/videos/73588-dieses-spiel-ist-fantastisch-cosmoteer`

### Category

breakage

### Describe the issue

Videos won't start with ublock activated since today.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.45.0
Firefox: 106
filterset (summary): 
  network: 119931
  cosmetic: 115888
  scriptlet: 18929
  html: 795
listset (total-discarded, last-updated): 
  added: 
    fanboy-annoyance: 82489-42837, 3d.22h.52m
    adguard-spyware: 36600-15834, 1d.22h.38m
    fanboy-cookiemonster: 42660-117, 3d.22h.53m
  default: 
    user-filters: 10-0, never
    ublock-filters: 31894-78, 1d.22h.37m
    ublock-badware: 3962-0, 1d.22h.36m
    ublock-privacy: 285-0, 3h.1m
    ublock-unbreak: 1906-0, 1d.22h.39m
    easylist: 64356-659, 3d.18h.50m
    easyprivacy: 30779-1718, 3d.18h.49m
    plowe-0: 3669-3, 12d.16h.6m
    DEU-0: 6020-2, 22h.53m
    ublock-abuse: 76-0, 3h
    urlhaus-1: 11744-0, 22h.54m
    ublock-quick-fixes: 505-0, 22h.55m
filterset (user): [array of 10 redacted]
trustedset: 
  added: [array of 96 redacted]
  removed: 
    edge-scheme
    wyciwyg-scheme
switchRuleset: 
  added: [array of 2 redacted]
hostRuleset: 
  added: [array of 11 redacted]
modifiedUserSettings: 
  advancedUserEnabled: true
  cloudStorageEnabled: true
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 3415 ms
  maxAssetCacheWait: 1470 ms
popupPanel: 
  blocked: 9
  blockedDetails: 
    pietsmiet.de: 3
    amazon-adsystem.com: 1
    getjad.io: 1
    google-analytics.com: 1
    googletagservices.com: 1
    imasdk.googleapis.com: 1
    outbrain.com: 1
```
</details>

**Comment by mapx-** [comment]
@Gamer10222 try installing the last update of uBO

https://github.com/gorhill/uBlock/releases/tag/1.45.1b1

Firefox: Click [uBlock0_1.45.1b1.firefox.signed.xpi](https://github.com/gorhill/uBlock/releases/download/1.45.1b1/uBlock0_1.45.1b1.firefox.signed.xpi)

**Comment by Gamer10222** [comment]
@mapx- Still the same.

**Comment by mapx-** [comment]
I created an account and ... still ok on my side.
Try disabling any other addon and test again.

**Comment by AntiAdvert** [comment]
- visit `about:addons`, is there [any other add-on which may block stuff](https://twitter.com/gorhill/status/1033706103782170625)?
- are you using any ad/tracker/etc.-blocking DNS service? (i believe that you can select dns services in the browser's settings, if it is up-to-date)
- are you using any modified [`hosts` file](https://www.thewindowsclub.com/hosts-file-in-windows)?

**Comment by Gamer10222** [comment]
@AntiAdvert Yes I have other addons installed but as defined in Prerequisites I did make sure to disable all of them, I'm using Quad9 as my default DNS, but since it works with UBlock disabled and I also tested it with Cloudflare DoH it does not seem to be related.
No my host file is not modified in a way that could lead to this error.

**Comment by Gamer10222** [comment]
@AntiAdvert Why did you mention #15611 ?

**Comment by AntiAdvert** [comment]
> @AntiAdvert Why did you mention this issue in #15611 ?

i linked a comment from `mapx-`
